pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -230,7 +230,7 @@ func init() {
 				},
 				{
 					APIGroups: []string{""},
-					Verbs:     sets.NewString("get", "list", "watch"),
+					Verbs:     sets.NewString("get", "list", "watch", "update"),
```

Comment:
this really seems like controlling the wrong layer :(
do we want to open an issue about it upstream?